.github/workflows/release.yml

```diff
@@ -33,6 +33,20 @@ jobs:
         name: windows-binary
         path: dist/newrelic_windows_amd64/newrelic.exe
 
+    - name: Checkout homebrew-core
```

Comment:
🤔💭🗒️: This could potentially be is own job with `needs: release` - e.g. `release-homebrew-formula-pull-request` or `release-homebrew-formula-pr` 

Thoughts?